chroma-manager/tests/integration/utils/test_blockdevices/test_blockdevice_lvm.py

```diff
@@ -20,27 +20,40 @@ def preferred_fstype(self):
     # Create a lvm on the device.
     @property
     def prepare_device_commands(self):
-        return ["vgcreate %s %s; lvcreate --wipesignatures n -l 100%%FREE --name %s %s" % (self.vg_name,
-                                                                                           self._device_path,
-                                                                                           self.lv_name,
-                                                                                           self.vg_name)]
+        return [
+            "vgcreate %s %s; lvcreate --wipesignatures n -l 100%%FREE --name %s %s"
+            % (self.vg_name, self._device_path, self.lv_name, self.vg_name)
+        ]
 
     @property
     def vg_name(self):
-        return "vg_%s" % "".join([c for c in self._device_path if re.match(r'\w', c)])
+        return "vg_%s" % "".join(
+            [c for c in self._device_path if re.match(r'\w', c)])
 
     @property
     def lv_name(self):
-        return "lv_%s" % "".join([c for c in self._device_path if re.match(r'\w', c)])
+        return "lv_%s" % "".join(
+            [c for c in self._device_path if re.match(r'\w', c)])
 
     @property
     def device_path(self):
         return "/dev/%s/%s" % (self.vg_name, self.lv_name)
 
     @classmethod
     def clear_device_commands(cls, device_paths):
-        return ["if vgdisplay %s; then vgremove -f %s; else exit 0; fi" % (TestBlockDeviceLvm('lvm', device_path).vg_name,
-                                                                           TestBlockDeviceLvm('lvm', device_path).vg_name) for device_path in device_paths]
+        lv_format = [
```

Comment:
These aren't really _format_ commands.  Can we change the variable name to be more accurate about what they really are, which is _destroy_, or _unconfigure_, or some such.

Reply:
Sure.